week-3/D-array-filter/exercise.js

```diff
@@ -8,7 +8,13 @@
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-var pairsByIndex; // Complete this statement
+var pairsByIndex = pairsByIndexRaw.filter(function(indexes) {
+  if (indexes !== null || indexes !== "string" || indexes !== "Boolean") {
```

Comment:
I'm not sure this is quite right. Does the string "whoops" match the string "string"? Do we instead need to check the type?